.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,77 @@
+name: GitHub CI
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: install deps
+        run: |
+          sudo apt-get update
+          sudo apt-get install cmake git ninja-build checkinstall
+          sudo apt-get install -y --no-install-recommends build-essential g++ gcc libass-dev pkg-config
+          git clone https://github.com/AviSynth/AviSynthPlus
+          cd AviSynthPlus
+          mkdir avisynth-build
+          cd avisynth-build
+          cmake ../ -G Ninja
+          ninja
+          sudo checkinstall --pkgname=avisynth --pkgversion="$(grep -r Version avs_core/avisynth.pc | cut -f2 -d " ")-$(date --rfc-3339=date | sed 's/-//g')-git" --backup=no --deldoc=yes --delspec=yes --deldesc=yes --strip=yes --stripso=yes --addso=yes --fstrans=no --default ninja install
+          cd ../..
+      - name: build
+        run: |
+          cmake -B build -S .
+          cmake --build build --clean-first
+      - name: copy
+        run: cmake -E copy "build/src/libassrender.so" "dist/libassrender.so"
+      - name: upload
+        uses: actions/upload-artifact@v2
+        with:
+          name: assrender_bin_linux
+          path: dist
+
+  build-win:
+    runs-on: windows-2019
+    steps:
+      - uses: actions/checkout@v3
+      - name: install submodules and nasm
+        run: |
+          git submodule update --init --recursive --remote
+          git clone https://github.com/ShiftMediaProject/VSNASM.git
+          .\VSNASM\install_script.bat
+      - name: revert freetype to VER-2-11-0 (libass windows bug)
+        run: |
+          cd SMP/freetype2
+          git checkout VER-2-11-0
+          cd ../..
```

Comment:
Why did you just add this? The submodule should already point at SMP’s (second) version of freetype’s 2.11.0 release according to your previous comment and the changelog iinm.